docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,769 @@
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+com.uid2.UID2Manager.Companion#init
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+see UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/Info.plist
+```
+
+</TabItem>
+</Tabs>
+
+You can change the credentials to ones assigned to you and connect to the UID2 Production environment if necessary. For details, see [Optional: Reduce Latency by Setting the API Base URL for the Production Environment](#optional-reduce-latency-by-setting-the-api-base-url-for-the-production-environment). (**GWH_SW are we still using the sample application here?**)
+
+## Complete UID2 Account Setup
+
+Complete the UID2 account setup by following the steps described in [Account Setup](../getting-started/gs-account-setup.md). As part of the account setup process, you'll need to provide a list of all the app names (Android Application ID, iOS App Bundle ID, and App Store ID) for the mobile apps that you'll be integrating with the UID2 mobile SDKs.
+
+When account setup is complete, you'll receive a [subscription ID and public key](../getting-started/gs-credentials.md#subscription-id-and-public-key). These values are unique to you, and you'll be required to use them to [configure the UID2 mobile SDK](#configure-the-uid2-mobile-sdk).
+
+## Add the UID2 Mobile SDK to Your Mobile App
+
+to add the mobile SDK to your app, follow the applicable documentation:
+
+- [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
+- [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
+
+(**GWH__SW not sure what it means "Follow the instructions on a draft version of a new UID2 SDK for Android reference guide" please clarify.**)
+
+At this point, you are ready to start generating UID2 tokens using the SDK.
+
+### Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment: [https://prod.uidapi.com](https://prod.uidapi.com). If you want to use the UID2 integration environment instead, provide the following URL in your call to initialize UID2Manager:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://operator-integ.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Bear in mind the following differences between environments:
+- Tokens from the UID2 integration environment are not valid for passing to the bid stream.
+- You'll have a different set of credentials (Subscription ID and public key) for each environment (integ and prod). Be sure to use the correct credentials for each environment.
```

Comment:
@genwhittTTD one thing we need to fix is remove the _credentials_ wording in these new docs - in other guides we don't call subscription ID/public key credentials but just "subscription id and public key) like [here](https://unifiedid.com/docs/guides/publisher-client-side#using-the-uid2-integration-environment)

Reply:
Oh dear @sunnywu . That credentials wording came entirely from me, I think. I thought it was a handy term. Are they not credentials? Let's touch on this in our meeting tonight so I understand better. But yes I will do this.

Reply:
@genwhittTTD subscription id and public key is a way to identify the caller so in that sense it is _credential_. but in JS/prebid.js CSTG guides, we don't use the term _credential_ at all so maybe we follow the same convention?

Reply:
Thank you Sunny.